team_docs/samples.md

```diff
@@ -0,0 +1,239 @@
+
+Guidelines for samples in documentation 
+--------------------------------------------------------
+The guideline provides a skeleton of what substitutes a good sample to the Kotlin signature. These are general recommendations, not strict rules. If ignoring some of the recommendations improves your documentation -- do that.
```

Comment:
"Kotlin signature" - I believe it's very unusual to use this term to describe functions, properties and classifiers, better to use more established "Kotlin declaration"

Reply:
End up with "Kotlin API entry"